css/dropdown.css

```diff
@@ -0,0 +1,44 @@
+ /* Dropdown Button */
```

Comment:
This file is an artifact that is no longer used and can be removed

Reply:
done